### ISSUE TITLE
left carousel-control and right carousel-control elements are not visible in the carousel example

### ISSUE THREAD

**Issue by bhutaninitin** [twbs/bootstrap#17878]
left carousel-control and right-carousel are not visible in http://v4-alpha.getbootstrap.com/examples/carousel/#. 

The carousel slides are vertically aligned. So, there should be "up carousel-control" and "down corousel-control" classes.

**Comment by heliumdream** [comment]
i was wondering about this myself, in the middle of coding a carousel site in bootstrap3, i decided i wanted to upgrade to version 4 to take advantage of relative units.

the handling of the carousel changed dramatically - notably the indicators have moved locations and changed from horozontal orientation to vertical.

even worse left and right arrow glyphs are gone!
after scrubbing the css i was losing my mind wondering where they were hiding...

perhaps [this](https://stackoverflow.com/questions/32612690/bootstrap-4-glyphicons-migration) may help explain things...Glyphicons front was dropped from bootstrap v4

**Comment by heliumdream** [comment]
after some more investigation, i have some insight and a fix. 

1) manually replace the fonts folder with the glyphs in it

2) jack a bunch of the glyph code from the old v3 bootstrap.css
notably @font-face, .glyphicon, and the many many individual glyph associations. 
and add this jacked code to the new v4 bootstrap.css (*I just tossed mine at the bottom)

3) after switching the css in the html from bootstrap.min.css back to bootstrap.css the arrow glyphs showed up again. hooray! the arrows appear.

*note: this does appear to a bug, and a difference between the minimzed and full version of the bootstrap css. I did glance at the .min code but couldn't identify the cause.

4) dismay! the arrows are now in the wrong location, and need to be updated to a responsive solution

I found a [stack thread](http://stackoverflow.com/questions/18516317/vertically-align-an-image-inside-a-div-with-responsive-height) with this [working fiddle solution](http://jsfiddle.net/hashem/46psK/) 

*there is a line in this solution that resets the font properties; i instead use font-size to control the size of the arrow, and i believe the situation inherits the proper responsive font and line-height properties from other parts of the bootstrap code. 

5) after appying this working solution to the v4 bootstrap.css and carousel html, the carousel looks as it did in v3.

afterthought: there was an [even better working fiddle solution](http://jsfiddle.net/hashem/46psK/4/), later in that stack thread. it handles case situations where images fall out of their container when the container shrinks too much.
